x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -435,11 +435,13 @@ const EventDetailsComponent: React.FC<Props> = ({
   const responseActionsTab = useResponseActionsView({
     rawEventData: rawEventData as RawEventData,
     ...(detailsEcsData !== null ? { ecsData: detailsEcsData } : {}),
-  });
+    isNewFlyout: false,
+  }) as EuiTabbedContentTab;
   const osqueryTab = useOsqueryTab({
     rawEventData: rawEventData as RawEventData,
     ...(detailsEcsData !== null ? { ecsData: detailsEcsData } : {}),
-  });
+    isNewFlyout: false,
+  }) as EuiTabbedContentTab;
```

Comment:
could probably avoid this cast if you make the return type of useOsqueryTab and useResponseActionsView to be EuiTabbedContentTab | undefined . Types being only layer of safety with eui makes casts extra dangerous